src/index.ts

```diff
@@ -69,6 +71,7 @@ export function throttling(octokit: Octokit, octokitOptions = {}) {
       clustering: connection != null,
       triggersNotification,
       minimumAbuseRetryAfter: 5,
+      minimumSecondaryRateRetryAfter: 5,
```

Comment:
I decided to keep both properties (`minimumAbuseRetryAfter` and `minimumSecondaryRateRetryAfter`) just in case but I'm not sure if it's necessary. 

Can they be overwritten with `...groups` somehow?